linera-execution/src/committee.rs

```diff
@@ -340,6 +343,12 @@ impl Committee {
             .map(|(name, validator)| (*name, validator.votes))
     }
 
+    pub fn fallback_keys_and_weights(&self) -> impl Iterator<Item = (AccountPublicKey, u64)> + '_ {
```

Comment:
Maybe `account_keys_and_weights`? (Better name it after what it is, not what it's currently used for.)

Reply:
Sure. I didn't use _account*_ b/c I don't like it that we have _account_ vs _validator_ b/c they're both belonging to a validator.

Reply:
Done in [Review part 1.](https://github.com/linera-io/linera-protocol/pull/3377/commits/edfb7bc5c4ef0158034e5d34cceffd76b9cba919).